Projects/l0/Sniffer/node_config.h

```diff
@@ -0,0 +1,65 @@
+#ifndef _NODE_CONFIG_H_
+#define _NODE_CONFIG_H_
+
+/*******************************************************************************
+ * PROJECT DEFINITION
+*******************************************************************************/
+
+/*******************************************************************************
+ * LUOS LIBRARY DEFINITION
+*******************************************************************************
+ *    Define                | Default Value              | Description
+ *    :---------------------|------------------------------------------------------
+ *    MAX_CONTAINER_NUMBER  |              5             | Service number in the node
+ *    MSG_BUFFER_SIZE       | 3*SIZE_MSG_MAX (405 Bytes) | Size in byte of the Luos buffer TX and RX
+ *    MAX_MSG_NB            |   2*MAX_CONTAINER_NUMBER   | Message number in Luos buffer
+ *    NBR_PORT              |              2             | PTP Branch number Max 8
+ *    NBR_RETRY             |              10            | Send Retry number in case of NACK or collision
+******************************************************************************/
+#define MAX_CONTAINER_NUMBER      1
+#define MSG_BUFFER_SIZE           2048
+#define MAX_MSG_NB                60
+
+#define PPTA_PORT				  0
+#define PPTA_PIN				  0
+#define PPTA_IRQ				  0
+#define PPTB_PORT				  0
+#define PPTB_PIN				  0
+#define PPTB_IRQ				  0
+#define NBR_PORT				  0
```

Comment:
put this redefine of Luos HAL pin under the description of HAL config define description and comment why